docs/extend/plugins_volume.md

```diff
@@ -16,67 +16,88 @@ keywords: "Examples, Usage, volume, docker, data, volumes, plugin, api"
 # Write a volume plugin
 
 Docker Engine volume plugins enable Engine deployments to be integrated with
-external storage systems, such as Amazon EBS, and enable data volumes to persist
-beyond the lifetime of a single Engine host. See the
+external storage systems such as Amazon EBS, and enable data volumes to persist
+beyond the lifetime of a single Docker ENgine host. See the
 [plugin documentation](legacy_plugins.md) for more information.
 
 ## Changelog
 
 ### 1.13.0
 
-- If used as part of the v2 plugin architecture, mountpoints that are part of paths returned by plugin have to be mounted under the directory specified by PropagatedMount in the plugin configuration [#26398](https://github.com/docker/docker/pull/26398)
+- If used as part of the v2 plugin architecture, mountpoints that are part of
+  paths returned by the plugin must be mounted under the directory specified by
+  `PropagatedMount` in the plugin configuration
+  ([#26398](https://github.com/docker/docker/pull/26398))
 
 ### 1.12.0
 
-- Add `Status` field to `VolumeDriver.Get` response ([#21006](https://github.com/docker/docker/pull/21006#))
-- Add `VolumeDriver.Capabilities` to get capabilities of the volume driver([#22077](https://github.com/docker/docker/pull/22077))
+- Add `Status` field to `VolumeDriver.Get` response
+  ([#21006](https://github.com/docker/docker/pull/21006#))
+- Add `VolumeDriver.Capabilities` to get capabilities of the volume driver
+  ([#22077](https://github.com/docker/docker/pull/22077))
 
 ### 1.10.0
 
-- Add `VolumeDriver.Get` which gets the details about the volume ([#16534](https://github.com/docker/docker/pull/16534))
-- Add `VolumeDriver.List` which lists all volumes owned by the driver ([#16534](https://github.com/docker/docker/pull/16534))
+- Add `VolumeDriver.Get` which gets the details about the volume
+  ([#16534](https://github.com/docker/docker/pull/16534))
+- Add `VolumeDriver.List` which lists all volumes owned by the driver
+  ([#16534](https://github.com/docker/docker/pull/16534))
 
 ### 1.8.0
 
-- Initial support for volume driver plugins ([#14659](https://github.com/docker/docker/pull/14659))
+- Initial support for volume driver plugins
+  ([#14659](https://github.com/docker/docker/pull/14659))
 
 ## Command-line changes
 
-A volume plugin makes use of the `-v`and `--volume-driver` flag on the `docker run` command.  The `-v` flag accepts a volume name and the `--volume-driver` flag a driver type, for example:
+To give a container access to a volume, use the `--volume` and `--volume-driver`
+flags on the `docker container run` command.  The `--volume` (or `-v`) flag
+works differently for plugins than for local volumes or bind mounts. It accepts
+a volume name and path within the container, and the `--volume-driver` flag a
+driver type.
+
+```bash
+$ docker container run -ti \
+    --volume volumename:/data \
+    --volume-driver=flocker \
```

Comment:
maybe we should use an example without the `--volume-driver` flag. That flag is a bit problematic, because you can only set a _single_ driver on `docker run`, so _every_ volume attached will use the same driver (including volumes defined in the Dockerfile / image).

Alternatively;

```
$ docker volume create --driver=flocker volumename

$ docker container run -it --volume volumename:/data busybox sh
```